packages/cloud_firestore/cloud_firestore/ios/Classes/FLTTransactionStreamHandler.m

```diff
@@ -11,12 +11,12 @@
 @interface FLTTransactionStreamHandler ()
 @property(nonatomic, copy, nonnull) void (^started)(FIRTransaction *);
 @property(nonatomic, copy, nonnull) void (^ended)(void);
+@property(strong) dispatch_semaphore_t semaphore;
```

Comment:
How come these were moved and made public public properties?

Reply:
If we didn't make them public, the compiler threw an error when trying to use `weakSelf` reference. but maybe that will change if we use strong reference outside the block.

Reply:
this error for example:
```
Property 'semaphore' not found on object of type 'FLTTransactionStreamHandler *
```